### DISCUSSION TITLE
Unable to Import Skins in Osu Lazer on iPad

### DISCUSSION THREAD

**Discussion by noelmiller** [ppy/osu#23057] [Q&A] [answered]
Hello everyone! 

**Tech Information**
iOS Version: 16.3.1
iPad Air 4th Generation
Osu Lazer Version (Through TestFlight): Version 1.0 (2023.326.1)

**Summary**
I have been struggling with getting skins imported and working on OSU Lazer on my iPad. I reviewed the GitHub issues and it appears that this feature has been supported for quite some time: https://github.com/ppy/osu/issues/7658

I have attempted opening the OSK file with the OSU app through the files menu, dragging it into the window, and nothing seems to get it to load in like it does on my PC. The skin I am trying to use is this one: https://osu.ppy.sh/community/forums/topics/1498493?n=1

Anybody have success with attempting to import skins on iPad? 

Thanks, 

Noel

**Comment by peppy** [comment, marked as answer]
It seems like this has regressed at some point – it doesn't work for me either.

**Reply by peppy** [reply]
Tracking via https://github.com/ppy/osu/issues/23106.

**Reply by noelmiller** [reply]
Hi Peppy! 

Thanks for opening an issue for this. I appreciate your work and looking forward to a fix! 

Thanks, 

Noel

**Comment by Zero001235** [comment]
In last update it works

**Comment by NothingwillbeNothing** [comment]
Hi, everyone

iPadOS Version: 18.2.1
iPad Air M2 11 inch
Osu Lazer Version (Sideloaded on my own): Version 1.0 (2025.118.3)

I have been trying to import osu! Skin, but no luck with atm. It must’ve been regressed at some point.
I have tried every possible ways to import like dragging into osu! App, sharing .osk file to osu, and nothing worked.
Can anyone confirm whether it’s only for me or not?

Thanks.
